lib/glimesh/api/resolvers/channel_resolver.ex

```diff
@@ -31,6 +31,26 @@ defmodule Glimesh.Api.ChannelResolver do
     end
   end
 
+  def update_stream_info(_parent, %{channel_id: channel_id, title: title},
+        %{context: %{access: access}
+      }) do
+    with :ok <- Bodyguard.permit(Glimesh.Api.Scopes, :stream_info, access) do
+      channel = Glimesh.ChannelLookups.get_channel(channel_id)
+      if channel !== nil do
```

Comment:
No reason to change this PR, but in the future you can use `with` for multiple checks. For example:
```
with :ok <- Bodyguard.permit(Glimesh.Api.Scopes, :stream_info, access),
      %Channel{} = channel <- Glimesh.ChannelLookups.get_channel(channel_id), 
      {:ok, changeset} <- Streams.update_channel(access.user, channel, %{title: title})  do
        # Your Happy Path
else
    # Any unhappy paths
end
```

Reply:
Alright. Is there anything else I need to change?

Reply:
No sorry, just need to find time personally to merge & deploy!